docs/import/manage_disaster.html

```diff
@@ -27,6 +27,11 @@
   <!-- Load our styles. -->
   <link rel="stylesheet" href="../style.css">
   <link rel="stylesheet" href="manage.css">
+  <style>
```

Comment:
can we stick this in a .css file somewhere?

Reply:
Went all in.